ui-v2/app/services/repository/type/event-source.js

```diff
@@ -38,20 +39,27 @@ const createProxy = function(repo, find, settings, cache, serialize = JSON.strin
               const status = get(e, 'errors.firstObject.status');
               if (status === '0') {
                 // Any '0' errors (abort) should possibly try again, depending upon the circumstances
+                // whenAvailable returns a Promise that resolves when the client is available
+                // again
+                return client.whenAvailable(e);
```

Comment:
This returns a Promise which resolves when the client (tab) becomes available again. As this is all based on a recurring Promise this essentially pauses the blocking query when the status of the error is zero (aborted) and then resolves the promise, and hence restarts the blocking query.